### ISSUE TITLE
Closing options-panel not working

### ISSUE THREAD

**Issue by a20edwth** [HGustavs/LenaSYS#12635]
This happens when trying to close the options-panel:

https://user-images.githubusercontent.com/102611046/169017620-8f57ea74-a742-469c-9656-15427790f926.mp4

**Comment by b20matad** [comment]
This issue was already fixed #12127 but it seems some old code got back in the merge.

**Comment by b19karhj** [comment]
To fix this issue do not remove the BGColor from toggleOptionsPane() as this will cause problem with the color slide not closing when you close the option panel. 
The cause of the problem is that BGColor is not created when you open and close the option panel without selecting anything.

I recommend this solution for the problem. As it checks only toggle BGColor if its not null, which will cause no error and will close the color panel if its open when you close the option panel.

Add this code to toggleOptionsPanel()

```
if(document.getElementById("BGColorMenu") != null){
            document.getElementById("BGColorMenu").style.visibility = "hidden";
        }
```
